src/specs/api.yaml

```diff
@@ -458,16 +458,27 @@ components:
           type: string
           example: GetEmbedding
           description: The name of this work -- must be `GetEmbedding`.
+        config:
+          type: object
+          description: The configuration settings to use for calculation of this embedding.
         type:
           type: string
           enum:
             - pca
             - tsne
             - umap
-          description: The type of embedding to compute.
+          description: The type of embedding to compute (deprecated; use config).
```

Comment:
You can add a deprecated tag for this in Stoplight Studio, it's a good idea because then we could check later for deprecated stuff being used.